bool is_master_ready

